docs/ExamplesContents.md

```diff
@@ -0,0 +1,63 @@
+# Terraform Provider Documentation: Examples
+This file lists all the examples in each category.
+
+## Clusters
+- [bastion](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/userdata/bastion.md)
+- [Mongodb](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/userdata/MongoDB.md)
+- [compute](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/compute.tf)
+- [datasources](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/datasources.tf)
+- [env-vars](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/env-vars)
+- [network](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/network.tf)
+- [outputs](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/outputs.tf)
+- [provider](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/provider.tf)
+- [README](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/README.md)
+- [variables](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/variables.tf)
+
+
+## Compute
+- [console_history](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/resources/core/console_history.md)
```

Comment:
I would expect this index to only point at examples under https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples.